docs/changelog.md

```diff
@@ -139,6 +139,11 @@ what we publish.
 
 - Added `StringSlice(..)` initializer from a `StringLiteral`.
 
+- Added a `byte_length()` method to `String`, `StringSlice`, and `StringLiteral`
+and deprecated their private `_byte_length()` methods. Added a warning to
+`String.__len__` method that it will return length in Unicode codepoints in the
```

Comment:
IIUC, this change doesn't actually do this, right? Looks like `String.__len__` just return the byte length right now.